rest-api-spec/src/main/resources/rest-api-spec/api/indices.delete.json

```diff
@@ -20,6 +20,11 @@
         "master_timeout": {
           "type" : "time",
           "description" : "Specify timeout for connection to master"
+        },
+        "include_hidden": {
```

Comment:
shouldn't this flag be added to many more rest spec (e.g. search)? Wherever we have expand_wildcards and co. I thought